packages/shorebird_cli/lib/src/shorebird_process.dart

```diff
@@ -106,7 +106,7 @@ class ShorebirdProcess {
     if (executable == 'flutter') {
       // If this ever changes we also need to update the `shorebird` shell
       // wrapper which downloads runs Flutter to fetch artifacts the first time.
-      return {'FLUTTER_STORAGE_BASE_URL': 'https://download.shorebird.dev/'};
+      return {'FLUTTER_STORAGE_BASE_URL': 'https://download.shorebird.dev'};
```

Comment:
The trailing slash results in `//` since flutter_tool already includes a `/` as part of the path.

Reply:
Is the system sensitive to this slash existing?  (If so we might want to fix that part of the system.).

Reply:
No I just noticed it while I was updating the proxy so I figured we'd remove it.